perl/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# Use the latest perl image from dockerhub
+FROM perl:latest
+
+# Install the web framework mojolicious
+RUN curl -L https://cpanmin.us | perl - -M https://cpan.metacpan.org -n Mojolicious
+
+# Instruct mojolicious to listen on port 8080 and open the port
+ENV MOJO_LISTEN http://*:8080
```

Comment:
I actually don't think that will work.  Today $PORT is defined in the base debian8 image :/